department-of-veterans-affairs/vets-website
# Check in/39273/update info

## Description
Rearranges pre-check-in confirmation page and adds accordions that show conditional messages based on demographics answers. Also contains a link to profile to update contact info.

To test:
() - Complete pre-check-in and verify that the confirmation page matches the new mock: https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/a/4a0voy3
() - Answer yes to all questions and confirm that the "How can I update my information?" accordion does not show.
() - Answer different yes/no answer confirmations and confirm the scenarios outlined here: https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/a/GmvYoDL

## Original issue(s)
department-of-veterans-affairs/va.gov-team#39273


## Testing done
Added unit test for new component and updated cypress tests.

## Screenshots
![localhost_3001_health-care_appointment-pre-check-in__id=46bebc0a-b99c-464f-a5c5-560bc9eae287 (6)](https://user-images.githubusercontent.com/13967174/167047658-dfc6db43-7da3-4355-af90-e9e0bfe07d50.png)


## Definition of done
